src/function/index.ts

```diff
@@ -176,12 +174,7 @@ function evaluateExponentialFunction(parameters, propertySpec, input) {
 
     const outputLower = parameters.stops[index][1];
     const outputUpper = parameters.stops[index + 1][1];
-    let interp = interpolate[propertySpec.type] || identityFunction; // eslint-disable-line import/namespace
-
-    if (parameters.colorSpace && parameters.colorSpace !== 'rgb') {
-        const colorspace = colorSpaces[parameters.colorSpace]; // eslint-disable-line import/namespace
-        interp = (a, b) => colorspace.reverse(colorspace.interpolate(colorspace.forward(a), colorspace.forward(b), t));
-    }
+    const i11nFn = interpolate[propertySpec.type] || identityFunction;
```

Comment:
Please change to full method name, without abbreviations.

Reply:
Oh I thought it was clever, interpolation => i11n just like internationalization is i18n :<
I will restore the previous name.

Reply:
Yea, I got it 😀 but still, I would even prefer the full name, not just interp.